plugins/catalog/src/components/EntityContextMenu/UnregisterEntity.tsx

```diff
@@ -72,5 +76,5 @@ export function UnregisterEntity(props: UnregisterEntityProps) {
     );
   }
 
-  return <>{unregisterButton}</>;
-}
+  return undefined;
```

Comment:
As long as we support react 17, we should never return undefined from components, only `null`. This changed in 18 afaik but we can't quite rely on that yet